src/stridelayout.jl

```diff
@@ -485,6 +492,17 @@ function strides(x)
         return Base.strides(x)
     end
 end
+
+# Fixes the example of https://github.com/JuliaArrays/ArrayInterface.jl/issues/160
+# TODO: Should be generalized to reshaped arrays wrapping more general array types
+function strides(A::ReshapedArray{T,N,P}) where {T, N, P<:AbstractVector}
+    if defines_strides(A)
+        return size_to_strides(size(A), first(strides(parent(A))))
```

Comment:
What about
```julia
julia> A = rand(5,12);

julia> B = reshape(A', (3,4,5));
```
`ArrayInterface.strides(B)` should be `(5,15,StaticInt{1}())`.

Reply:
I would prefer to fix that later since I would need to think a bit more about it. I opened https://github.com/JuliaArrays/ArrayInterface.jl/issues/162 to track this bug.